charts/cosigned/templates/webhook/webhook_validating.yaml

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ template "cosigned.fullname" . }}-webhook-tls
+  name: {{ template "cosigned.fullname" . }}-validating
+  labels:
+    {{- include "cosigned.labels" . | nindent 4 }}
+    control-plane: {{ template "cosigned.fullname" . }}-webhook
+webhooks:
+  - clientConfig:
+      service:
+        name: {{ template "cosigned.fullname" . }}-webhook
+        namespace: {{ .Release.Namespace }}
+        path: /validations
+    failurePolicy: Ignore
+    #namespaceSelector:
```

Comment:
Given the scope of the types of resources that are affected by this policy, there has to be a configurable way to limit the blast radius

Reply:
did an initial customization